File: main/src/models/NewServiceModel.ts
import {Employee, Hall, RentedItem, Service} from "./Models";

declare type NewServiceModel = {
    id: number
    service: Service,
    startDateTime?: Date,
    duration?: number,
    hall?: Hall,
    employee?: Employee,
    rentedItem?: RentedItem,
    number?: number,
    isFullTime?: boolean,
}

export default NewServiceModel;

File: WebServer/WebServer.ASP/Dto/NewServiceModel.cs
using Microsoft.EntityFrameworkCore;
using PhotostudioDB;
using PhotostudioDB.Models;

namespace WebServer.ASP.Dto;

public class NewServiceModel
{
    public int ServiceId { get; set; }
    public DateTime? StartDateTime { get; set; }
    public int? Duration { get; set; }
    public int? HallId { get; set; }
    public int? EmployeeId { get; set; }
    public int? RentedItemId { get; set; }
    public int? Number { get; set; }
    public bool? IsFullTime { get; set; }
}